docs/content/usage/actions/comparison.en-us.md

```diff
@@ -118,18 +112,6 @@ Pre and Post steps don't have their own section in the job log user interface.
 
 ## Different behavior
 
-### Downloading actions
-
-Gitea Actions doesn't download actions from GitHub by default.
-"By default" means that you don't specify the host in the `uses` field, like `uses: actions/checkout@v3`.
-As a contrast, `uses: https://github.com/actions/checkout@v3` has specified host.
-
-The missing host will be filled with `https://gitea.com` if you don't configure it.
-That means `uses: actions/checkout@v3` will download the action from [gitea.com/actions/checkout](https://gitea.com/actions/checkout), instead of [github.com/actions/checkout](https://github.com/actions/checkout).
-
-As mentioned, it's configurable.
-If you want your runners to download actions from GitHub or your own Gitea instance by default, you can configure it by setting `[actions].DEFAULT_ACTIONS_URL`. See [Configuration Cheat Sheet](administration/config-cheat-sheet.md#actions-actions).
-
```

Comment:
Should we instead mention (or link) to the config docs, since this does now download from GH by default unless set to `self` or fully-qualified?